services/magnetomer.md

```diff
@@ -0,0 +1,24 @@
+# Magnetometer
+
+    identifier: 0x13029088
+    extends: _sensor
+    camel: magneto
+
+A 3-axis magnetometer.
+
+## Registers
+
+    ro forces @ reading {
+        x: i16.16 F
+        y: i16.16 F
+        z: i16.16 F
+    }
+
+Indicates the current magnetic field on magnetometer.
+
+## Commands
+
+    command calibrate @ calibrate
```

Comment:
need a register to expose calibration status: calibration needed, calibrating, calibrated
+ event to notify that calibraiton changed

Reply:
or send the "calibrated" register update when it changes